src/lib/forgotPassword.js

```diff
@@ -0,0 +1,6 @@
+import axios from './axios';
+
+export const userForgotPassword = async email => {
+  const forgotPassword = await axios.post('/forgot-password', { email });
+  return forgotPassword;
+};
```

Comment:
Can you use a try/catch here, since you are doing some asynchronous operations here?